docs/app/pages/components/components-sections/dashboard/widgets/dashboard-widgets.component.html

```diff
@@ -0,0 +1,171 @@
+<ux-dashboard #dashboard
+              class="customizable-dashboard"
+              [class.dashboard-grabbing]="dashboard.isGrabbing"
+              [options]="options">
+
+    <ux-dashboard-text-widget
+        class="widget"
+        id="widget-text"
+        [name]="'Text Widget'"
+        [heading]="'Text Widget'"
+        [fixedMode]="fixedMode"
+        [text]="loremIpsum"
+        [editable]="textWidgetEditable"
+        [colSpan]="2"
+        [rowSpan]="2">
+    </ux-dashboard-text-widget>
+
+    <ux-dashboard-table-widget
+        class="widget"
+        id="widget-table"
+        [name]="'Table Widget'"
+        [heading]="'Table Widget'"
+        [fixedMode]="fixedMode"
+        [header]="tableHeader"
+        [data]="tableData"
+        [colSpan]="2"
+        [rowSpan]="2">
+    </ux-dashboard-table-widget>
+
+    <ux-dashboard-actions-widget
+        class="widget"
+        id="widget-actions"
+        [name]="'Actions Widget'"
+        [heading]="'Actions Widget'"
+        [fixedMode]="fixedMode"
+        [status]="status"
+        [actions]="actions"
+        [colSpan]="2"
+        (actionSelected)="handleActionChange($event)">
```

Comment:
Any string properties should not use bindings but instead just pass them as a static value, e.g.

```html
name="Table Widget"
heading="Table Widget"
```

There are a few other files that do this that also should be updated.